tools/combine_metaphlan2_humann2/.shed.yml

```diff
@@ -0,0 +1,6 @@
+categories: 
+  - Metagenomics 
+description: Combine MetaPhlAn2 and HUMAnN2 outputs to relate genus/species abundances and gene families/pathways abundances
+name: combine_metaphlan2_humann2 
+owner: bebatut
```

Comment:
@bebatut we would need to have admin rights for those repos.

Reply:
we can change there the ownership.... happy to transfer the ownership